### PR TITLE
Create tr.md

### PR DESCRIPTION
**Pull request by selek55**

### CODE REVIEW

**Comment by selek55** [comment]
signed a CLA

**Comment by medyagh** [comment]
@selek55 unfortunately the CLA is not signed yet or the system hasn't recognized it. would you please verify you signed the CLA with same email as `git config`?

**Comment by selek55** [comment]
signed a CLA

**Comment by spowelljr** [comment]
@selek55 Thank you for signing the CLA! We have unfortunately updated the template for our GitHub Issues since your contribution. Would you mind updating it to match the new version? https://raw.githubusercontent.com/kubernetes/minikube/master/.github/ISSUE_TEMPLATE/__en-US.md

File: .github/ISSUE_TEMPLATE/tr.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: Türkçe
+about: Sorun bildirin
+---
```
Comment (spowelljr) [inline .github/ISSUE_TEMPLATE/tr.md:4]: Could you please add a third tag here.
`labels: l/tr`

**Review by spowelljr — CHANGES_REQUESTED** [review]

**Comment by spowelljr** [comment]
@selek55 I'm going to close this issue due to it being open for a long duration. If at a later date you find the time to address the comments, feel free to reopen, thanks!

**Pull request closed without merging**